change/react-native-windows-66f4660e-05be-46b7-8321-74ea94fb2b4b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
[putting the convo here so we get threading]

> > > Question: from this snapshot and inspecting component properties from VS, it seems like the visibility remains as "visibile," despite the behavior changing as if it was collapsing(and from setting breakpoints in FrameworkElementViewManager I can see that the right code is getting hit and the Visibility value gets set to "Collapsed" - but I never see this in XAML) - any idea what's going on?
> >
> > if the tree dump says it's not collapsed then it's probably not collapsed : ) (and something else may be re-setting the property?)
> Do you see the collapsed element in the UI? What does the Live Visual Tree and Property panes say?
> 
> The "collapsed" element is no longer seen in the UI, but the Live Visual Tree and accompanying properties do not seem to change when display:none is toggled. If I undo the code change I added, same thing happens - component goes out of view when display is set to none, but you are still able to tab into those components - when I add the change back in, I'm no longer able to tab into components which are set to display:none, but according to the properties the visibility is still "visible"...

Reply:
Sometimes you need to refresh the live tree view - does it remain collapsed after you 'f5' in VS and refresh the inspector?

Reply:
There is currently a bug in the live visual tree/property pane where it won't refresh: https://developercommunity.visualstudio.com/t/Live-visual-tree-properties-do-not-refre/1359952

Given that, the only trustworthy option right now seems to be to look at the xaml tree in WinDBG -  we have a debugger extension that lets you do that, we can follow up offline

Reply:
Confirmed that's it's collapsing 👍 but filed #7392 as a separate issue for testing.. removing snapshots for now, I'll add those back in once that gets fixed.